app/test_main.py

```diff
@@ -1 +1,41 @@
-# write your code here
+import pytest
+from unittest import mock
+from app.main import cryptocurrency_action as ca
```

Comment:
I'm not sure it's a great alias for function. I would suggest not to use alias at all.